### ISSUE TITLE
{"status": 401, "error_message": "Requires authentication", "title": "invalid_token", "error_type": "invalid_token", "detail": "Requires authentication", "type": "https://quay.io/api/v1/error/invalid_token"}

### ISSUE THREAD

**Issue by koudaiii** [koudaiii/qucli#32]
## WHY

- homebrew

```shell-session
$ qucli get koudaiii/qucli
Repository:
	quay.io/koudaiii/qucli
Visibility:
	public
Permissions:
	koudaiii(admin)
Notifications:
err: HTTP error!
URL: https://quay.io/api/v1/repository/koudaiii/qucli/notification
status code: 401
body:
{"status": 401, "error_message": "Requires authentication", "title": "invalid_token", "error_type": "invalid_token", "detail": "Requires authentication", "type": "https://quay.io/api/v1/error/invalid_token"}
```

- manual go build

```
$ ./bin/qucli get koudaiii/qucli
Repository:
	quay.io/koudaiii/qucli
Visibility:
	public
Permissions:
	koudaiii(admin)
Notifications:
	b0736be9-f0eb-4c3c-8d33-2e331b1e5b0f	Some title	repo_push	map[]	slack	map[url:https://hooks.slack.com/service/some/token/here]
	00ad6dc9-8a20-47c2-85c2-17a5bbecd918	Some title	repo_push	map[]	slack	map[url:https://hooks.slack.com/service/some/token/here]
```

🤔 ❓ 

qucli version: 0.6.2

**Comment by koudaiii** [comment]
```
$ qucli get koudaiii/qucli
Repository:
	quay.io/koudaiii/qucli
Visibility:
	public
Permissions:
	koudaiii(admin)
Notifications:
	b0736be9-f0eb-4c3c-8d33-2e331b1e5b0f	Some title	repo_push	map[]	slack	map[url:https://hooks.slack.com/service/some/token/here]
```


ok, Fix [v0.6.4](https://github.com/koudaiii/qucli/releases/tag/v0.6.4)

**Comment by akshitdevops** [comment]
@koudaiii When I try to get the tags of an image , I get the same issue can you please suggest how to get the tags of an image in quay.io (via commandline)
